lang/ja.json

```diff
@@ -680,15 +680,15 @@
       "actions": {
         "resetPassword": {
           "label": "パスワード変更",
-          "playerPrompt": "パスワードをリセットしたいアカウントの名前を入力してください",
+          "playerPrompt": "パスワードをリセットしたいアカウントの名前を入力してください。",
```

Comment:
Why are we only changing this one? There are other player prompts without a period.

Reply:
All playerPrompts have been changed.
How about this?